### ISSUE TITLE
Improve interlock between implementation SIGs and Spec/SemConv 

### ISSUE THREAD

**Issue by svrnm** [open-telemetry/sig-contributor-experience#9]
* How can we improve collaboration?
* How can we make it easier for implementing SIGs to address their needs?
* How can we help the Spec/SemConv to find SIGs that provide reference implementation?

**Comment by codefromthecrypt** [comment]
As a new member, things feel like a normal monolith problem. For example, semantics are managed as one big unit, releases included. This helps from having very large count of dependencies and CODEOWNERS or repos, but it does feel more limited  than a delegated ecosystem. Please excuse/correct inaccuracies as I'm new.

For example, right now there's no publication (I'm aware of) of emerging semantics, so you have to consume libraries that include all of them. This is usually monthly but it can be longer than that. It is hard to test changes prior to release and different than ecosystems that have like a main/snapshot etc build of their outputs.

On review side, the mechanics are also monolithic, while certainly friendly towards feedback. The main glitch I see there is that domain experts have an informal relationship with approvals on the content they would know best. That this works is just a function of being listened to.

For me as an implementor, I mostly run into this chicken egg problem. Specs should really follow practice if you think about it, but there is no connection between the specs and experimentation except how specific individuals choose to behave. For example, you can modify something and get a PR reviewed and merged with not even a single implementor trying it. This is currently as I understand not even a surprise. However, from an implementor POV it feels off. I would love to see a way for a playground to exist with emerging SIGs so that even very very early ideas can routinely be put to code. This can prevent accidents and put more signal into decisions.

Hope the brain dump helps!

**Comment by svrnm** [comment]
@codefromthecrypt your comment might be more related to #11, as this here is more an issue for maintaer (or approver) level collaboration

**Comment by codefromthecrypt** [comment]
@svrnm I'm not sure if you commented on the issue you meant to? #11 is about recognition right? Maybe I didn't use the correct words, but I thought I was discussing collaboration things from various angles.

perhaps my point "domain experts have an informal relationship with approvals on the content they would know best" came off sounding like recognition ask, but actually I was probably unclear about what I meant by "domain expert".

So for example, the LLM SIG dispatches changes of files to a few people here https://github.com/open-telemetry/community/blob/c6f5a02f409a4c79d0cf80d07f17cd45330cf552/config.yaml#L1093-L1098

I am probably in the minority on this point, but I think a SIG about a topic area would be even better if more domain experts, such key end users, very active maintainers of frequently instrumentation concerns like openai top 5 contributor, LLM author etc.

Even if talking about that file, probably some clarity about how to get onto that file. For example, I keep pinging implementors manually as there are several other vendors besides those in the file. Maybe there's a way we that even if only a few people should be in that file, others can more easily subscribe to changes that only affect one SIG.

Basically, it is more about the notifications, right now a select few will be maintainers or approvers. Understood. When there are many others with equal or even more stake, it isn't clean to subscribe to a SIG's file changes like CODEOWNERS does. You end up doing things like watching everything and make mail filter rules.

It would be nice to both make it easier for people who don't have privs to be notified, and once that happens possibly a good time to focus on recruiting domain experts, too. That way things happen automatically, not because someone remembered to ping a non-approver.

Hope this makes more sense?

**Comment by svrnm** [comment]
That file you are referring to is a work in progress document without any implications right now. It is a pure snapshot of the teams that exist already, here is the merged PR for context:

https://github.com/open-telemetry/community/pull/2210

This is part of the SIG Project Infra work that just started a few weeks/days back, so please ignore that for the time being.

On the topic itself: I agree that what you describe is an issue between SemConv and Implementors, and reading through that I recognize that we might need to split this issue here into "Interlock between implementing SIGs and Spec" and "interlock between implementors and SemConv" since while related those problems require different solutions.

**Comment by amy-super** [comment]
@svrnm I'm inclined to close this issue as it hasn't had a lot of progress made, and I think other parts of the project have addressed the needs described here. Any last interest here before we close?
